Cargo.toml

```diff
@@ -22,6 +22,7 @@ webpki = "0.21.3"
 rustls-pemfile = "1.0.0"
 hex = "0.4.3"
 tokio = { version = "1.7.1", features = ["fs"] }
+tracing = { version = "0.1", features = ["log"] }
```

Comment:
Do you happen to know how much this prolongs compile times? If yes would it be worth it to make it optional?

Reply:
I don't know.  And I'm not exactly certain how to make it optional. =/

I'm actually 100% happy to pull the tracing out of here.  It seems like something it *should* have but me just shoving it in to do some small changes is probably not ideal.  It would be better if it were done everywhere and consistently.

Reply:
Thinking about it, `tracing` has MSRV 1.49 which could become annoying since I want MSRV 1.48 so yeah, making it optional in a separate PR would be the best appraoch.

Making it optional should be very easy, just set `optional = true` in `Cargo.toml` and then use `#[cfg_attr(feature = "tracing", tracing::instrument(...))] for attributes and

```rust
#[cfg(feature = "tracing")]
{
    tracing::trace!("message");
}
```

for logs in the code.